index.html

```diff
@@ -0,0 +1,194 @@
+          <li>
+            <i class="fab fa-facebook-f" aria-hidden="true"></i>
+          </li>
+          <li>
+            <i class="fab fa-twitter" aria-hidden="true"></i>
+          </li>
+          <li>
+          <li>
+            <h4>English</h4>
+          </li>
+          <li>
+            <h4>My Page</h4>
+          </li>
+          <li>
+            <h4>Logout</h4>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="secondary-navbar">
+      <div class="secondary-sub-container">
+        <div class="navbar-logo">
+          <a class="font-lato" href="index.html">CCS SUMMER CONCERT <br> Series 2021</a>
+        </div>
+        <!-- open and close icons -->
+        <i class="fas fa-bars fa-3x text-white cursor-pointer" id="openMenu" aria-hidden="true" ></i>
+
+        <nav class="pages-menu" id="mob-nav">
+          <i class="fa fa-times fa-3x text-white" aria-hidden="true" id="closeMenu"></i>
+          <ul class="navbar-items">
+            <li class="page-sections font-lato home-link">
+              <a href="index.html" class="active-tab">Home</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="about.html">About</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">Program</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">Join</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="page-sections font-lato">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="product">
+            <a class="font-lato" href="index.html">CCS Concert</a>
+          </div>
+        </nav>
+      </div>
+    </div>
+  </header>
+
+  <section class="first-section">
+    <div class="overlay">
+      <div class="first-section-container">
+        <h2 class="open-intro-heading font-lato">"Hello Kids!, It's time to have fun"</h2>
+        <h1 class="open-heading">Little Kids at the Summer Concert Series 2021</h1>
+        <div class="first-section-para">
+          <p class="para font-lato">
+            A joyful celebration bringing kids together to celebrate togetherness and forget everything else,<br>A child has a special way of adding joy to everyone. The kids who want the most love, will ask for it in the most unloving way. Sometimes we have to go deep inside ourselves to solve our problems. And so a live concert is exciting because of all the electricity that is generated in the crowd and on stage. It is the favourite part of the live concerts.
+          </p>
+        </div>
+        <div class="event-details">
+          <h3>2021.09.11(SAT) ~ 12(SUN)</h3>
+          <p class="venue font-lato">
+            @ National Theatre, Kampala, Uganda.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] The typography and line texts of your _hero_ section do not match the provided [desgn template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Once again, we actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a _top_ section with a column that includes a **small line of text**, **the page title**, a **block of text**, a **medium line of text in bold letters**, and a **smaller line of text**, as can be seen in the provided template design. Let's ensure that the **font family**, font sizes, **font weights**, **font colors**, and **letter-spacing** matches that of the provided template design

**Your hero design:**
![image](https://user-images.githubusercontent.com/12745474/132957296-3a9d0dae-9bcc-4ec8-90e7-13356bba807e.png)


**The original hero design:**
![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)